hydra/utils.py

```diff
@@ -51,6 +67,22 @@ def get_method(path: str) -> Callable[..., Any]:
 get_static_method = get_method
 
 
+def get_object(path: str) -> Any:
+    """
+    Look up a callable based on a dotpath.
```

Comment:
the comment he is incorrect. object may not be a callable.

Reply:
I've opened PR #2544 with a fixup.